activerecord/test/cases/attribute_test.rb

```diff
@@ -5,7 +5,6 @@ module ActiveRecord
   class AttributeTest < ActiveRecord::TestCase
     setup do
       @type = Minitest::Mock.new
-      @type.expect(:==, false, [false])
```

Comment:
This is no longer needed since `CallbackChain#default_terminator` now does not check if the returned value is `false`.